css/style.css

```diff
@@ -0,0 +1,683 @@
+    align-items: center;
+  }
+
+  .last-text {
+    display: none;
+  }
+
+  .par {
+    font-size: calc(6px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+  }
+
+  body {
+    padding: 0;
+  }
+
+  .container-fluid {
+    padding-right: 10px;
+    padding-left: 10px;
+    overflow: hidden;
+  }
+
+  .image-mag-2 {
+    width: 100%;
+  }
+
+  .cover {
+    padding-top: 15px;
+  }
+
+  .h-6-1 {
+    font-size: 10px;
+  }
+
+  .h-6-2 {
+    font-size: 10px;
+  }
+
+  .h-6-logo {
+    font-size: calc(6px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+    line-height: 3px;
+    color: #ef4430;
+    font-weight: 700;
+  }
+
+  .h-7-nav {
+    font-size: calc(7px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+  }
+
+  nav li {
+    margin-left: -5px;
+  }
+}
+
+@media only screen and (max-width: 994px) {
+  .bottom-header {
+    display: none;
+  }
+
+  .iphone {
+    display: none;
+  }
+
+  .top-story {
+    height: 1830px;
+    overflow-y: scroll;
+  }
+
+  .icon-header-1 {
+    display: none;
+  }
+
+  .logo-mobile {
+    width: 50%;
+    padding-right: 4px;
+  }
+
+  .header {
+    height: 60px;
+    margin-bottom: 1px;
+    display: grid;
+    grid-template-columns: 3.2fr 0.5fr 2fr;
+    grid-template-rows: 1fr;
+  }
+
+  .top-header {
+    display: grid;
+    grid-template-columns: 3.2fr 0.5fr 2fr;
+    grid-template-rows: 1fr;
+    grid-column: 1 / 4;
+    grid-row: 1 / 2;
+    height: 60px;
+    background-color: #ef4430;
+    clip-path: polygon(0% 0%, 100% 0%, 100% 100%, 0% 100%);
+  }
+
+  .mobile {
+    display: grid;
+    grid-column: 1 / 2;
+    grid-row: 1 / 2;
+  }
+
+  .mobile-cont {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .h-6-logo {
+    font-size: calc(6px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+    line-height: 3px;
+    color: #ef4430;
+    font-weight: 700;
+  }
+
+  .under-button-logo {
+    height: 20px;
+    width: 90%;
+  }
+
+  .third-cont {
+    padding-right: 10px;
+  }
+
+  .logo-1 {
+    display: none;
+  }
+
+  .icon-header {
+    display: none;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  .icon-arrow-logo-2 {
+    font-weight: 900;
+    font-size: 12px;
+    color: white;
+  }
+
+  .icon-header-2 {
+    color: white;
+    font-size: calc(9px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+    font-weight: 900;
+  }
+
+  .icon-arrow-logo-sub {
+    font-size: calc(6px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+  }
+
+  .h-6-1-logo {
+    font-size: 12px;
+    height: 25px;
+    padding-right: 5px;
+  }
+
+  .icon-burger {
+    display: inline;
+    padding-left: 5px;
+    color: #fff;
+    font-weight: 900;
+    font-size: calc(18px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+  }
+
+  .footer-top {
+    flex-direction: column;
+  }
+
+  .img-fluid {
+    width: 100%;
+  }
+}
+
+@media only screen and (max-width: 765px) {
+  .top-story-1 {
+    display: inline;
+  }
+
+  .top-story {
+    display: none;
+  }
+}
```

Comment:
You should also rely on bootstrap classes for [responsive break-points](https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints).